.ci/ci_cache_components.sh

```diff
@@ -59,7 +60,10 @@ cache_kernel_artifacts() {
 cache_cloud_hypervisor() {
 	local cloud_hypervisor_binary="cloud-hypervisor"
 	local cloud_hypervisor_binary_path="${GOPATH}/src/github.com/cloud-hypervisor/cloud-hypervisor/cloud-hypervisor/"
+	go get -d "${runtime_repo}" || true
```

Comment:
the `get_version` function manages this, so this shouldn't be needed.
https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L142

Reply:
@chavafg changes applied thanks